distributions/debian/rules

```diff
@@ -14,6 +14,14 @@ endif
 %:
 	dh $@
 
+override_dh_update_autotools_config:
+	# dh_update_autotools_config replaces libs/opus/config.{sub,guess}.
+	# This is unnecessary as we don't build opus via autotools at all
+	# (we use qmake). In addition, this causes our -dev version generation
+	# logic to mark Debian builds as -dirty by default.
```

Comment:
I'd add a link to this PR here

Reply:
Hm, why do that here? We don't add Github Issue/PR links for other code parts either. If you think that something is missing in the comment, I'm happy to add it. If someone wants to track that back to Github, this should be possible using `git blame` and `git log`.

Reply:
You describe an issue which is fixed. "In addition ... by default." Therefore I thought it would make sense to linke this PR for further information. 

You could also change the grammar: ... this causes --> This would have caused...

Reply:
Ok, I've added the *would* part in the hope that it makes it clearer. The point of the comment is to state that there are two reasons:
- First, the config.guess replacement behavior is unnecessary, as the file is not used at all.
- Second (therefore "in addition"), it causes the `-dirty` behavior.